modules/artifact/src/main/java/org/mule/module/artifact/classloader/ErrorHooksConfiguration.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.artifact.classloader;
```

Comment:
I believe the same note from DefaultResourceReleaser applies here. Should I replicate it in the class javadoc?